Source/Particles/Collision/BinaryCollision/PhotonPhoton/SinglePhotonPhotonCollisionEvent.H

```diff
@@ -0,0 +1,183 @@
+/* Copyright 2021 Neil Zaim
```

Comment:
Same comment as above:
```suggestion
/* Copyright 2023 Arianna Formenti
```